docs/docs/migrate-from/facebook-wit-ai-to-rasa.mdx

```diff
@@ -23,12 +26,12 @@ rasa init
 ```
 
 This will create a directory called `data`.
-Remove the files in this directory, and
-move the expressions.json file into this directory.
+Remove the files in this directory, and move the content of the `utterances` directory
+to `data`.
 
 ```bash
-rm -r data/*
-mv /path/to/expressions.json data/
+rm -rf data/
+mv /path/to/utterances data/
```

Comment:
but `/data` won't exist given that you've `rm -rf`-ed it in the line before 😛

Reply:
this way it would create the directory and move it's content to `data/`
(you can treat is as renaming — we are basically renaming `/path/to/utterances` directory here to `data`)